packages/shopify-codemod/test/fixtures/assert/falsy-called-method-to-assert-not-called/basic.output.js

```diff
@@ -0,0 +1,2 @@
+assert.neverCalledWith(foo, bar, qux);
```

Comment:
The `called-method-to-assert-called` handles similar cases.  Nothing like `assert.notOK(foo.calledTwice)` appears in our code, so I didn't expend any effort on it.  

Which assertion would provide the most meaningful message for that scenario?  `assert.notEquals(foo.callCount, 2)`?